app/lib/actions/katello/repository/destroy.rb

```diff
@@ -81,6 +83,11 @@ def handle_acs_product_removal(repository)
           end
         end
 
+        def delete_orphaned_filters(repository)
```

Comment:
```suggestion
        def delete_empty_filters(repository)
```

Reply:
Both don't accurately reflect which filters are being deleted. Thoughts on this:  `delete_empty_repo_filters` ? Also we should change the name of param of controller if we change this since that is user facing.